src/pages/security/vulnerabilities-types/index.js

```diff
@@ -0,0 +1,174 @@
+import React from 'react'
+import styled from 'styled-components'
+import Hero from '../_hero'
+import { vulnerabilitiesTypesContent } from '../static/_vulnerabilities-types-content'
+import { localize, WithIntl, Localize } from 'components/localization'
+import Layout from 'components/layout/layout'
+import { Container, Flex, SEO } from 'components/containers'
+import { Header, Text } from 'components/elements'
+import device from 'themes/device'
+
+const StyledContainer = styled(Container)`
+    width: 79.2rem;
+    padding: 80px 16px;
+
+    @media ${device.tablet} {
+        width: auto;
+    }
+`
+
+const CardWrap = styled.div`
+    border-radius: 4px;
+    background-color: var(--color-grey-30);
+    margin-top: 16px;
+    padding: 8px 0;
+    display: inline-flex;
+`
+
+const CardItem = styled.div`
+    border-left: ${({ second_item }) => (second_item ? 'solid 1px var(--color-grey-6)' : 0)};
+    border-right: ${({ second_item }) => (second_item ? 'solid 1px var(--color-grey-6)' : 0)};
+    padding: 0 24px;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+`
+
+const CardText = styled(Text)`
+    margin-top: 4px;
+
+    @media ${device.tablet} {
+        font-size: 12px;
+        line-height: 18px;
+    }
+`
+
+const CardSmallText = styled(Text)`
+    @media ${device.tablet} {
+        font-size: 10px;
+        line-height: 14px;
+    }
+`
+
+const MediumText = styled(Text)`
+    @media ${device.tablet} {
+        font-size: 14px;
+        line-height: 20px;
+    }
+`
+
+const SmallText = styled(Text)`
+    @media ${device.tablet} {
+        font-size: 12px;
+        line-height: 18px;
+    }
+`
```

Comment:
You can use the`tablet` prop for passing in an CSS style object into `Text` for the responsive styles.